### PR TITLE
[MINI-5639] Improve test code coverage 

### PR DESCRIPTION
**Pull request by shohiebsense**
# Description
Also Ignoring MiniAppConfig.kt

## Links
[MINI-5635](https://jira.rakuten-it.com/jira/browse/MINI-5635)

# Checklist
- [x] I have read the [contributing guidelines](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](https://github.com/rakutentech/android-miniapp/blob/master/.github/CONTRIBUTING.md#sdk-development-learning-path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `./gradlew check` without errors

### CODE REVIEW

**Comment by shohiebsense** [comment]
Added more cases, codecov not detecting the newer ones...(?)



Screenshot 2022-10-20 100604

**Review by gopi-karmakar — COMMENTED** [review]
@shohiebsense 
As we discussed kindly separate the multiple verifications in to multiple test cases instead clubbing in to one. Also find new ideas to add more test cases let's say: Verifying the if else block of code or verifying the returning messages or verifying the thrown exceptions type and messages.

**Comment by shohiebsense** [comment]
> @shohiebsense As we discussed kindly separate the multiple verifications in to multiple test cases instead clubbing in to one. Also find new ideas to add more test cases let's say: Verifying the if else block of code or verifying the returning messages or verifying the thrown exceptions type and messages.

So Insightful 💡 , thanks a lot, would do

File: miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt [hunk 1]
```diff
@@ -32,11 +32,18 @@ interface MiniAppFileChooser {
...
+     * @param isCaptureEnabled derived from WebChromeClient function
+     * @param fileChooserParamsAcceptTypes derived from WebChromeClient function
      **/
+    @Suppress("LongParameterList")
     fun onShowFileChooser(
         filePathCallback: ValueCallback<Array<Uri>>?,
         fileChooserParams: WebChromeClient.FileChooserParams?,
-        context: Context
+        context: Context,
+        isOpenMultipleMode: Boolean = fileChooserParams?.mode == WebChromeClient.FileChooserParams.MODE_OPEN_MULTIPLE,
```
Comment (munir-rakuten) [inline miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt:44]: Good work, just one suggestion,i think we don't need these parameters, as `onShowFileChooser` is a public interface, we should not add parameters that has no external use, please try refactor the code the way that we dont have to add extra parameters just for test case, thanks

**Review by munir-rakuten — COMMENTED** [review]
Good work, just one suggestion added.

**Review by laubengaier — APPROVED** [review]
lgtm, ton of work - good job! 👏 

only remark on my side here is the PR is quite heavy and contains a lot of syntax styling fixes (styling can be done separately) tests could have been split into multiple PR's with the relating features eg `[MINI-5639] Improve test code coverage / SecureStorage` `[MINI-5639] Improve test code coverage / FilePicker` reducing the review time for us and you can get faster feedback loops

**Review by munir-rakuten — APPROVED** [review]
Well done.

**Comment by shohiebsense** [comment]
Many thanks @munir-rakuten @laubengaier 

agree that would be ideal, will keep it in mind.  Not gonna lie it was soul-consuming, but I learned so much as the leverage. :D

**Pull request merged**